can/interfaces/socketcan/socketcan.py

```diff
@@ -44,7 +38,7 @@
     LimitedDurationCyclicSendTaskABC,
 )
 from can.typechecking import CanFilters
-from can.interfaces.socketcan.constants import *  # CAN_RAW, CAN_*_FLAG
+from can.interfaces.socketcan.constants import *  # pylint: disable=unused-wildcard-import
```

Comment:
Maybe, we should just inline that file?

Reply:
We could, but i think that might break downstream code. How about using the constants namespace?
`from can.interfaces.socketcan import constants`